examples/docker-build/dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM node:10-stretch-slim as builder
+
+WORKDIR /src
+
+# Remove unnecessary files from node_modules (.md, .ts, ...)
+RUN cd /tmp && \
+  curl -#L https://github.com/tj/node-prune/releases/download/v1.0.1/node-prune_1.0.1_linux_amd64.tar.gz | tar -xvzf- && \
+  mv -v node-prune /usr/local/bin && rm -rvf * && \
+  echo "node-prune" > /usr/local/bin/node-clean && chmod +x /usr/local/bin/node-clean
+
+COPY package*.json ./
+RUN npm install --no-progress && node-clean
+
+# Build app
+COPY . .
+RUN npm run build
+
```

Comment:
After building, it would better idea to strip all `devDependencies` and only keep `dependencies` (prod) for less image size and reducing the attack surface.

Reply:
Something like this:

```
RUN rm -rf node_modules && \
 NODE_ENV=production yarn install \
  --prefer-offline \
  --pure-lockfile \
  --non-interactive \
  --production=true
 ```